charts/camunda-platform-alpha/test/unit/packages/testutils.go

```diff
@@ -0,0 +1,73 @@
+package testutils
+
+import (
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/helm"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	corev1 "k8s.io/api/core/v1"
+)
+
+type ConfigMapTest struct {
+    suite.Suite
+    ChartPath string
+    Release   string
+    Namespace string
+    Templates []string
+}
+
+type TestCase struct {
+    Name     string
+    Values   map[string]string
+    Expected map[string]string
+}
+func TestCreateTestSuite(t *testing.T, templates []string) {
+    t.Parallel()
+
+    chartPath, err := filepath.Abs("../../../")
+    require.NoError(t, err)
+
+    suite.Run(t, &ConfigMapTest{
+        ChartPath: chartPath,
+        Release:   "camunda-platform-test",
+        Namespace: "camunda-platform-" + strings.ToLower(random.UniqueId()),
+        Templates: templates,
+    })
+}
+
+func (s *ConfigMapTest) RenderTemplate(options *helm.Options) corev1.ConfigMap {
+    output := helm.RenderTemplate(s.T(), options, s.ChartPath, s.Release, s.Templates)
+    var configmap corev1.ConfigMap
+    helm.UnmarshalK8SYaml(s.T(), output, &configmap)
+    return configmap
+}
+
+func (s *ConfigMapTest) VerifyConfigMap(testCase string, configmap corev1.ConfigMap, expectedValues map[string]string) {
+    for key, expectedValue := range expectedValues {
+        actualValue := strings.TrimSpace(configmap.Data[key])
+        s.Require().Equal(expectedValue, actualValue, "Test case '%s': Expected key '%s' to have value '%s', but got '%s'", testCase, key, expectedValue, actualValue)
+    }
+}
+
+func (s *ConfigMapTest) RunTestCases(testCases []TestCase) {
```

Comment:
I wonder if running tests this way makes it harder to run a single test in isolation. such as hitting the play button next to a test in an IDE or running `go test` from the CLI

Reply:
Ups, I forget to remove this package actually - was my first test and it did not work this way - removed this. The actual package I created was testhelper.go - sorry for that.

The testhelper.go keeps the test suite.Run in the test file and with that isolates it better. Agree with your comment!